dbms/src/test/java/org/polypheny/db/cypher/AggregateTest.java

```diff
@@ -105,15 +111,14 @@ public void doubleCountRenameAggregateTest() {
 
 
     @Test
-    @Ignore // own min max impl necessary
```

Comment:
Can you enable this test?
However, something seems to be wrong with this class anyway, since this @Ignore is Junit 4... This should thrown an error.

Reply:
This seems to be an error with github, as this method is already enabled and there is no `@Ignore` anymore.

Reply:
This change removes `@Ignore`.